jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
 layout: classic-docs
 title: "Language Guide: Python"
```

Comment:
Improve this title for SEO

Reply:
How would you like me to improve this title?